docs/src/developer/developer/coding-style.md

```diff
@@ -0,0 +1,49 @@
+
+# The wire-server Coding Style Guide
+
+This document contains a collection of rules as they emerge from
+day-to-day discussions.  If you are unhappy with anything, open a PR
+and trigger a discussion.  Coding style PRs of course should be
+reviewed by more or less the entire team if feasible.
+
+These rules are by no means enforced consistently at the time of
+writing them down, but they are offer a way to avoid debate when
+reviewing current pull requests: if there is more than one way of
+doing things, whoever can point to a rule in this document wins the
+argument.
+
+Where feasible and if there is time, rules should be enforced
+automatically.
+
+### Integer literals
+
+Integer literals are spelled `50_000` (not `50 * 1000` and not
+`50000`).  (NB: ghc allows to write things like `40_0000`, which makes
+sense eg. in Chinese.  This is illegal in this code base for obvious
+reasons.)
```

Comment:
```suggestion
sense eg. in Chinese.  However, this codebase requires underscores
after every 3 digits.)
```

Might not be very obvious a Chinese person (or a South Asian person)

Reply:
I tried to find the name of this millions, billions, trillions numeral system, but I couldn't.